api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst

```diff
@@ -0,0 +1,170 @@
+.. _put-update-identity-provider-v2.0:
+
+Update IDP
+~~~~~~~~~~
+
+.. code::
+
+   PUT /v2.0/RAX-AUTH/federation/identity-providers/{identityProviderId}
+
+Update an Identity provider.
```

Comment:
Identity provider (IDP). (You are correct not to use the abbreviation in a title. You just define it in its first occurrence in the body.)